src/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php

```diff
@@ -386,11 +386,26 @@ protected function getConfig(string $name, $default = false): string
      */
     protected function getVersion(): string
     {
-        $tempVersion = $this->tempVersion ?? '';
-        if (!$tempVersion) {
-            $this->tempVersion = $this->getConfig('VERSION');
+        $version = $this->tempVersion ?? '';
+
+        if ('' !== $version) {
+            return $version;
```

Comment:
I think it's overkill because, any kind of not false will reflect a valid result so we only need `!$version`.

Reply:
In my opinion it's easier for debugging purposes. What value _can_ `$version` even have theoretically? As far as I understood, it can only be unset or an empty string, it's never `false`, `null` or any other falsey value (other than `''`). 

Since it can only be unset or a string, I think this makes the most sense.

Reply:
Ok. I ask ChatGPT which version is more clean code.
```php
A)
$version = $this->tempVersion ?? '';
if ('' !== $version) {
return $version;
}

B)
$version = $this->tempVersion ?? '';
if (!$version) {
return $version;
}
```
> ChatGPT: From a clean code perspective, variant A is clearer and preferable. Here are the reasons
> Explicit comparisons: Variant A uses an explicit comparison ('' !== $version) which makes it clear that the variable is not empty. Variant B, on the other hand, uses a non-explicit comparison (if (!$version)) which may be less understandable for less experienced developers because there are several values ​​that evaluate to false (empty strings, the number 0, null, false).
>
> Readability: An explicit comparison increases the readability of the code because it is clear what exactly is being checked. This makes it easier for other developers to understand and maintain the code.
>
> Avoiding confusion: In PHP, the expression if (!$version) can easily be misunderstood because it covers many possible values ​​(empty string, null, false, 0). This can lead to confusion if it is not clear which values ​​tempVersion can take.

So the reasons were very good. So I'm convinced of A.